tests/Traits/MocksApiCommunication.php

```diff
@@ -36,6 +36,8 @@ protected function getClientMock()
                     $filePath = preg_replace('/.json/', '.txt', $filePath);
                 }
 
+                $filePath = preg_replace('/\-page\-(number|size)\-\-[0-9]*/', '', $filePath);
```

Comment:
Removing page number and size from the url means we cannot test pagination anymore, this should still be here.